lib/ansible/modules/identity/cyberark/cyberark_user.py

```diff
@@ -187,7 +198,6 @@ def user_details(module):
             msg=("Unknown error while performing user_details."
                  "\n*** end_point=%s%s\n%s" % (api_base_url, end_point, to_text(unknown_exception))),
             headers=headers,
-            exception=traceback.format_exc(),
```

Comment:
keep these (and below)

Reply:
Moved to https://github.com/ansible/ansible/pull/52515